### PR TITLE
Dr J's (maybe not) last post

### PR DESCRIPTION
**Pull request by zmustapha**
I've completed my edit. Feel free to make more edits if I missed a few.

### CODE REVIEW

File: content/articles/eof.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
... the beginning of the millennium I found myself backpacking across Eastern Europe as a software developer, participating in coding hackathons held in Viennese castles and Norwegian fjords. Intermingling with artists and social theorists was an essential component of these gatherings. I wasn’t particularly politically conscious as an undergrad, and my experiences within a vibrant open sources community awakened my interest in what some might call politics — governance, decision making and conflict resolution. I also came to understand the Plone community as a self-organized knowledge community, one that continually promotes active learning among all participants.  I would’ve not read Seymour Papert’s  [_Mindstorms_](https://en.wikipedia.org/wiki/Mindstorms_%28book%29) until years later, but looking back it is easy to recognize how this open source community began to fulfill Papert’s vision of schools without walls or classrooms. I also experienced the lessons of Jacques Rancière’s [_The Ignorant Schoolmaster_](https://en.wikipedia.org/wiki/Jacques_Ranci%C3%A8re#Life_and_work Ignorant Schoolmaster) first-hand as the Python community collectively embarked on intellectual emancipation.
```
Comment (nikolas) [inline content/articles/eof.md]: s/open sources/open source/

File: content/articles/eof.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+I decided to plot a course to the academy, unsure how I would make the transition. I applied to the Instructional Technology masters program at Teachers College and while my application was under review I was hired a developer at CCNMTL; and the rest is history. The job came with free tuition, and my original plans flipped as I began working full-time at CCNMTL and studying part-time. CCNMTL’s executive director, [Frank Moretti](http://alchemicalmusings.org/2013/07/16/dear-frank/), was also on the faculty of Teachers College and the Columbia Graduate School of Journalism’s doctoral program in communication. He soon became  my mentor, boss, teacher and friend.
```
Comment (nikolas) [inline content/articles/eof.md]: s/I was hired a developer/I was hired as a developer/

File: content/articles/eof.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ This is my final post on this blog as a Columbia employee. Taking a cue from our
...
... the beginning of the millennium I found myself backpacking across Eastern Europe as a software developer, participating in coding hackathons held in Viennese castles and Norwegian fjords. Intermingling with artists and social theorists was an essential component of these gatherings. I wasn’t particularly politically conscious as an undergrad, and my experiences within a vibrant open sources community awakened my interest in what some might call politics — governance, decision making and conflict resolution. I also came to understand the Plone community as a self-organized knowledge community, one that continually promotes active learning among all participants.  I would’ve not read Seymour Papert’s  [_Mindstorms_](https://en.wikipedia.org/wiki/Mindstorms_%28book%29) until years later, but looking back it is easy to recognize how this open source community began to fulfill Papert’s vision of schools without walls or classrooms. I also experienced the lessons of Jacques Rancière’s [_The Ignorant Schoolmaster_](https://en.wikipedia.org/wiki/Jacques_Ranci%C3%A8re#Life_and_work Ignorant Schoolmaster) first-hand as the Python community collectively embarked on intellectual emancipation.
```
Comment (nikolas) [inline content/articles/eof.md]: I want to note that these long lines make these diffs very hard to read. When I write hugo posts I limit the line length in the markdown files to 80 characters.

Comment (thraxil) [inline content/articles/eof.md]: Semantic linefeeds? http://rhodesmill.org/brandon/2012/one-sentence-per-line/

File: content/articles/eof.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+title: "EOF"
+topics:
+- Projects
+type: post
+---
+
+This is my final post on this blog as a Columbia employee. Taking a cue from
+our not-so-lame-duck prez (see:
+[#obamajama](http://jama.jamanetwork.com/article.aspx?articleid=2533698
+#obamajama)), I’ll indulge in the conceit of reflecting on my legacy at
```
Comment (nikolas) [inline content/articles/eof.md]: It looks like this link: `[#obamajama](http://jama.jamanetwork.com/article.aspx?articleid=2533698#obamajama))` needs to be on a single line to fix formatting.

Comment (nikolas) [inline content/articles/eof.md]: Also - `#obamajama` should be removed from the end of the link. The URL is just http://jama.jamanetwork.com/article.aspx?articleid=2533698

**Pull request merged**